Tests/AppKitTests/NSViewExtensionsTests.swift

```diff
@@ -123,6 +123,18 @@ final class NSViewExtensionsTests: XCTestCase {
 		view.removeSubviews()
 		XCTAssertEqual(view.subviews.count, 0)
 	}
+    
```

Comment:
These are trailing white spaces. You should get SwiftLint warnings about these. You can change your Xcode settings to clear them for you. In your Xcode properties go to the **Text Editing** section and enable **Automatically trim trailing whitespace**, along with **Including whitespace-only lines** as well.